README.md

```diff
@@ -31,18 +31,16 @@ For news and announcements follow us on twitter [@peekabooAV](https://twitter.co
 
 ### Installation ###
 
-Clone the repository
+Install from PyPI into a new virtual environment:
 ```shell
-git clone https://github.com/scVENUS/PeekabooAV.git
+virtualenv --python=python3 /path/to/venv
```

Comment:
Wouldn't it be better to use `python3 -m venv /path/to/venv`?
It doesn't require an additional installed package and doesn't copy the python binary to the venv.

Reply:
Interesting you should ask that because I was thinking about that as well but wasn't sure about the current best-practice. I had already converted all the documented calls to `python3 -mvenv` when I noticed that we still need to create a python2 venv for Cuckoo. My understanding is that with `-mvenv` the venv gets the interpreter version of the python interpreter it's called with and isn't available on python2. So we'd still need to install a `virtualenv` tool for creation of the Cuckoo environment. (A quick glance at the first paragraph of the virtualenv documentation at https://virtualenv.pypa.io/en/latest/ seems to confirm that now.)

I reverted then all calls to `virtualenv` for consistency, i.e. not confuse users (and ourselves in a couple of days time) with two different ways to achieve seemingly the same thing.

BTW: The `virtualenv` tool is cross-python-version-capable, i.e the one from package `python3-virtualenv` is running under python3 but capable of creating python2 venvs. So we only need one version of it to create venvs for multiple python interpreter versions.

Uh, I just noticed that it's not the `python3-virtualenv` package we need but `virtualenv` which on Ubuntu 18.04 at least pulls in `python3-virtualenv` and provides the actual `/usr/bin/virtualenv` binary.

Reply:
I've updated the patch with a longish explanation of why we tell the user to install just the `virtualenv` package. When I looked at how the installer might benefit from that I realised that ansible is written in python itself. A quick check reveals that it's forced to python2 on Ubuntu 18.04. I have no idea if it calls the `virtualenv` binary or imports the module from `python-virtualenv`. My guess is that either way it only works currently because we have Peekaboo and Cuckoo on the same host and therefore always both the binary and the python2 module installed. It'll likely break as soon as we support separating those roles. I vote to just remember that piece of fact when it comes to that instead of right now trying to get this right without a reliable way of testing it.